openai-audio.el

```diff
@@ -49,14 +50,15 @@ Argument FILE is audio file to transcribe, in one of these formats: mp3, mp4,
 mpeg, mpga, m4a, wav, or webm.  CALLBACK is the execuation after request is
 made.
 
-Arguments BASE-URL, CONTENT-TYPE, KEY and ORG-ID are global options; however,
+Arguments BASE-URL, PARAMETERS, CONTENT-TYPE, KEY and ORG-ID are global options; however,
```

Comment:
I prefer max 80 column per line, can you fix that for me? (And all other places)

Reply:
done